networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_snippets.py

```diff
@@ -178,3 +178,5 @@
 REMOVE_GLOBAL_CONFIG = """
             <cmd>no %s</cmd>
 """
+
```

Comment:
can you add some comments here then? All other commands in this file have comments also so.